### ISSUE TITLE
Buttercup login page automatically open in new tab during Chrome startup

### ISSUE THREAD

**Issue by janpelikan** [buttercup/buttercup-browser-extension#159]
Hello,
during every Chrome launch Buttercup automatically open in new tab for Unlock archive. It's kinda annoying, at least for me. Is it possible add an option to enable or disable this as a optional feature?
Thank you.

**Comment by perry-mitchell** [comment]
Of course, I think this is a fair addition. We currently assume the most-common use-case is that when the browser is opened, the user will want to log in to their password manager. We auto-open the login page by default, but will provide a setting to disable this soon.

**Comment by perry-mitchell** [comment]
Blocked by settings: #160

**Comment by Zjemm** [comment]
it would be even more appealing if this would not open a new tab, but just a popup or dropout from the buttercup icon, like enpass for example.

ill track this project because i find it very nice. but its not mature enough for me for my daily work.....but it will get there

**Comment by th3littl3minds** [comment]
> it would be even more appealing if this would not open a new tab, but just a popup or dropout from the buttercup icon, like enpass for example.

I would greatly appreciate this feature too! I'm introducing my team to buttercup but they all disabled the extension because of the annoying tab opening every time they launch chrome.

1Password also does this with a dropout from the extension icon and you're not bothered by it until you try to login somewhere

**Comment by perry-mitchell** [comment]
At least everything I read online specifies that it isn’t possible to force the extension’s pop up menu to open.. so is there another way you’d recommend?

I’m open to changing its behaviour until we get the settings page implemented.

**Comment by th3littl3minds** [comment]
> At least everything I read online specifies that it isn’t possible to force the extension’s pop up menu to open.. so is there another way you’d recommend? I’m open to changing its behaviour until we get the settings page implemented.

In my opinion the popup menu doesn't have to open automatically.

I really like 1Password's behaviour:
![password-manager-behaviour](https://user-images.githubusercontent.com/36683323/45550691-edf28a80-b82b-11e8-9cd2-7e8ee3bf11ff.png)

1. If you click on the icon within the input field, a popup opens that tells you to press the [shortcut](https://developer.chrome.com/extensions/commands) to unlock the password manager.
2. The shortcut opens the password manager's login popup which lets you unlock the manager for this session
3. If the manager is already unlocked, the shortcut opens the password manager

Hope this helps!

**Comment by perry-mitchell** [comment]
Thanks for the detailed suggestion. This actually aligns well with another issue we have regarding unlocking vaults, and I like how this would make the process a bit easier for some.

I personally prefer the automatic opening of the page, however. Lastpass also does this for 2FA. I think a combination of the login prompt within the in-window menu and the option to enable/disable the automatic opening of the page might be necessary..

**Comment by th3littl3minds** [comment]
> I think a combination of the login prompt within the in-window menu and the option to enable/disable the automatic opening of the page might be necessary..

Yeah, the combination of those options would probably be the best solution. 
I just got used to pressing `ctrl` + `shift` + `x` whenever I need to login to the password manager.

**Comment by perry-mitchell** [comment]
This will be configurable in the upcoming 2.0 release.

**Comment by th3littl3minds** [comment]
Awesome! looking forward to 2.0 😃

**Comment by perry-mitchell** [comment]
This is done, and is again working in 2.2.0
